packages/database/src/migrations/20201111013312-RemoveItemsFromLineChartsInUnfpaDashboard-modifies-data.js

```diff
@@ -0,0 +1,191 @@
+import { updateBuilderConfigByReportId } from '../utilities/migration';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+// Table name = 'Reproductive Health Product Average Monthly Consumption (AMC), Country Name'
+const UNFPAReproductiveHealthProductAmcDashboardReport = {
+  id: 'UNFPA_Reproductive_Health_Product_AMC',
+  targetItems: {
+    UNFPA_RH_AMC_SAYANA_Press: {
+      codes: ['AMC_4752843e'],
+      color: '#8C5AFB',
+      label: 'SAYANA Press',
+      legendOrder: 8,
+    },
+    UNFPA_RH_AMC_Norethisterone: {
+      codes: ['AMC_542a34bf'],
+      color: '#FD6AC4',
+      label: 'Norethisterone',
+      legendOrder: 9,
+    },
+    UNFPA_RH_AMC_Implant_Contraceptives: {
+      codes: ['AMC_3ff944bf'],
+      color: '#0F7F3B',
+      label: 'Implant Contraceptives',
+      legendOrder: 4,
+    },
+  },
+};
+
+// Table name = 'Reproductive Health Products Months of Stock (MOS) (mSupply data)'
+const UNFPAReproductiveHealthProductMosNationalDashboardReport = {
+  id: 'UNFPA_Reproductive_Health_Product_MOS_National',
+  targetItems: {
+    UNFPA_RH_MOS_Medroxyprogesterone_acetate_104mgper0dot65ml_SAYANA_Press: {
+      codes: ['MOS_4752843e'],
+      color: '#8C5AFB',
+      label: 'SAYANA Press',
+      legendOrder: 8,
+    },
+    UNFPA_RH_MOS_Norethisterone_enantate_200mgpermL_in_1mL_ampoule_oily_solution: {
+      codes: ['MOS_542a34bf'],
+      color: '#FD6AC4',
+      label: 'Norethisterone',
+      legendOrder: 9,
+    },
+    UNFPA_RH_MOS_Etonogestrel_releasing_implant_single_rod_containing_68mg_of_etonogestrel: {
+      codes: ['MOS_3ff944bf'],
+      color: '#0F7F3B',
+      label: 'Implant',
+      legendOrder: 4,
+    },
+  },
+};
+
+// Table name = 'Reproductive Health Products Months of Stock (MOS)'
+const UNFPAReproductiveHealthProductMosDashboardReport = {
+  id: 'UNFPA_Reproductive_Health_Product_MOS',
+  targetItems: {
+    UNFPA_RH_MOS_Medroxyprogesterone_acetate_104mgper0dot65ml_SAYANA_Press: {
+      codes: ['MOS_4752843e'],
+      color: '#8C5AFB',
+      label: 'SAYANA Press',
+      legendOrder: 8,
+    },
+    UNFPA_RH_MOS_Norethisterone_enantate_200mgpermL_in_1mL_ampoule_oily_solution: {
+      codes: ['MOS_542a34bf'],
+      color: '#FD6AC4',
+      label: 'Norethisterone',
+      legendOrder: 9,
+    },
+    UNFPA_RH_MOS_Etonogestrel_releasing_implant_single_rod_containing_68mg_of_etonogestrel: {
+      codes: ['MOS_3ff944bf'],
+      color: '#0F7F3B',
+      label: 'Implant',
+      legendOrder: 4,
+    },
+  },
+};
+
+// Table name = 'Reproductive Health Stock Status: Months Of Stock (mSupply)'
+const UNFPARhProductsMosDashboardReport = {
+  id: 'UNFPA_RH_Products_MOS',
+  targetItems: {
+    UNFPA_RH_SAYANA_Press_MOS: {
+      codes: ['MOS_4752843e'],
```

Comment:
I suggest breaking these up into dataBuilderConfig and viewJson for clarity. I figured it out when I scrolled down to the implementation but it took me a minute, at first I thought one of us might have had old data in their database and I went to check.

Reply:
Hi Igor, thanks for reviewing this. Sorry about the one minute headache💆🏻‍♂️. I will break it up now.